pom.xml

```
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>edge-sip2</artifactId>
-  <version>2.0.1-SNAPSHOT</version>
+  <version>2.0.2-SNAPSHOT</version>
```

Per the community [guidelines](https://dev.folio.org/guidelines/release-procedures/), master should be a minor release ahead, unless breaking changes are introduced requiring a major release. We don't want to introduce new features as a patch (bugfix) release.
```suggestion
  <version>2.1.0-SNAPSHOT</version>
```

If we need a subsequent 2.0.2 release, we would do the work in a branch created from the 2.0.1 tag.